controller/schema.go

```diff
@@ -271,6 +271,21 @@ $$ LANGUAGE plpgsql`,
 		$$`,
 		`UPDATE apps SET meta = jsonb_merge(meta, '{"flynn-system-critical":"true"}') WHERE name IN ('discoverd', 'flannel', 'postgres', 'controller')`,
 	)
+	migrations.Add(14,
+		`CREATE TYPE backup_status AS ENUM ('running', 'complete', 'error')`,
```

Comment:
I think we're using join tables instead of enums. Take a look at the migrations above for examples.